Makefile

```diff
@@ -101,7 +104,12 @@ uninstall: manifests kustomize
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) webhook paths="./..." output:crd:artifacts:config=$(CRD_BASES)
```

Comment:
Might be good to make ocp version as a var so that it can be used for future release updates.
There was an example in 4.6 release: https://github.com/openshift/sriov-network-operator/blob/release-4.6/Makefile#L111

Reply:
Done, thanks